packages/state-transition/src/block/processBlsToExecutionChange.ts

```diff
@@ -0,0 +1,43 @@
+import {capella, ssz} from "@lodestar/types";
+import {BLS_WITHDRAWAL_PREFIX, ETH1_ADDRESS_WITHDRAWAL_PREFIX, WITHDRAWAL_PREFIX_BYTES} from "@lodestar/params";
+import {toHexString, byteArrayEquals} from "@chainsafe/ssz";
+import {digest} from "@chainsafe/as-sha256";
+
+import {CachedBeaconStateCapella} from "../types.js";
+
+export function processBlsToExecutionChange(
```

Comment:
Please add whitespace in the function body. 1 new line after each logic group, for better readability